src/shared/components/login/facebook.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import config from 'config/environment';
+import FacebookLogin from 'react-facebook-login';
+import SocialLogin from './socialLogin';
+import styles from './socialMediaButtons.css';
+
+class Facebook extends Component {
```

Comment:
Apologies for not catching this earlier, but both `facebook.js` and `google.js` can be stateless, functional components

`const Facebook = () => { ... }`